packages/slate-react/test/index.spec.tsx

```diff
@@ -22,16 +22,16 @@ describe('slate-react', () => {
       test('should not unmount the node that gets split on a split_node operation', async () => {
         const editor = withReact(createEditor())
         const value = [{ type: 'block', children: [{ text: 'test' }] }]
-        const mounts = jest.fn<void, [Element]>()
+        const mounts = jest.fn()
 
         let el: ReactTestRenderer
 
         act(() => {
           el = create(
             <Slate editor={editor} value={value} onChange={() => {}}>
               <Editable
-                renderElement={({ element, children }) => {
-                  React.useEffect(() => mounts(element), [])
+                renderElement={({ children }) => {
+                  useEffect(() => mounts(), [])
```

Comment:
This is meant to be testing the mount call count (hence the empty dependency) but we were passing the element which eslint wants included in the dependency array. It's not needed for the test.